pkg/storaged/filesystem/mounting-dialog.jsx

```diff
@@ -152,7 +152,7 @@ export function mounting_dialog(client, block, mode, forced_options) {
         }
 
         function maybe_lock() {
-            if (mode == "unmount") {
+            if (mode == "unmount" && !subvol) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19690-20240111-133844-24a81545-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)